api/src/main/scala/Geoprocessing.scala

```diff
@@ -216,7 +221,8 @@ trait Geoprocessing extends Utils {
     */
   private def rasterGroupedCount(
     rasterLayers: Seq[TileLayerCollection[SpatialKey]],
-    multiPolygon: MultiPolygon
+    multiPolygon: MultiPolygon,
+    opts: Options
```

Comment:
Do we need to do this for RasterLinesJoin too? I assume not since that's just for MapShed, but figured it was worth verifying.

Reply:
Only Polygonal and Extent rasters take Options (see https://github.com/locationtech/geotrellis/blob/7bd79361838b28b724b51ad5e8f7cf2344b070f6/raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala#L94-L121), which is why RasterLinesJoin does not have it.